pom.xml

```diff
@@ -14,6 +14,7 @@
         <module>service</module>
         <module>web</module>
         <module>dao</module>
+        <module>http_client</module>
```

Comment:
Also, it's OK, just a little nit: for modules\artifacts `kebab-case` is preferable

Reply:
Refactored